openeo_processes_dask/process_implementations/__init__.py

```diff
@@ -26,3 +26,6 @@
 import rioxarray as rio  # Required for the .rio accessor on xarrays.
 
 import openeo_processes_dask.process_implementations.cubes._xr_interop
+
+log_format = "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+logging.basicConfig(level=logging.INFO, format=log_format)
```

Comment:
Never call `logging.basicConfig` (or any other logging setup code) from the top-level of a module.
That's the responsibility of an application's entrypoint (something like `main()`).

Doing it from here can not be undone easily and which is frustrating for the users of you library

Reply:
Ah, very good point, thanks so much for pointing this out, this is super useful!!